scripts/install-snowcli.sh

```diff
@@ -1,24 +1,20 @@
-﻿#!/bin/sh
+#!/usr/bin/bash
 
 set -euo pipefail
-PIPX_PATH="snow_pipx_path"
 
 # These commands ensure that each time `snow` command is executed the system will use 
 # the executable in the pipx installation folder and not in any other installation folder.
-if [ -z "${PIPX_BIN_DIR}" ]; then
-    export PIPX_BIN_DIR="~/.local/bin/$PIPX_PATH"
-else 
-    export PIPX_BIN_DIR="$PIPX_BIN_DIR/$PIPX_PATH"
-fi
 
-mkdir -p "$PIPX_BIN_DIR"
+PIPX_BIN_DIR=${PIPX_BIN_DIR:-"~/.local/bin"}
```

Comment:
`FOO=${BAR:-"foz"}` is "if BAR is not set, use foz as value". [More information.](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html)